### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by marusak**
I haven't tested this locally because I don't have `zanata-client` anymore. I'll try it with bots and if that fails and I need to debug it then I spin out locally task container.

 * [x] po-refresh

### CODE REVIEW

**Comment by cockpituous** [comment]
po-refresh in progress on centosci-tasks-w32g0.
Log: https://logs-https-cockpit.apps.ci.centos.org/logs/po-refresh-229-20191024-100505/

**Comment by cockpituous** [comment]
Command '('bots/make-source', 'update-po')' returned non-zero exit status 2.: https://logs-https-cockpit.apps.ci.centos.org/logs/po-refresh-229-20191024-100505/

**Comment by cockpituous** [comment]
po-refresh in progress on 3-cockpit-9.
Log: https://209.132.184.41:8493/logs/po-refresh-229-20191024-112727/

**Comment by cockpituous** [comment]
Command '['make', 'upload-pot', 'clean-po', 'download-po']' returned non-zero exit status 2.: https://209.132.184.41:8493/logs/po-refresh-229-20191024-112727/

**Comment by cockpituous** [comment]
po-refresh in progress on centosci-tasks-bmx66.
Log: https://logs-https-cockpit.apps.ci.centos.org/logs/po-refresh-229-20191024-114848/

**Comment by cockpituous** [comment]
po-refresh  done: https://github.com/cockpituous/cockpit-podman/commits/po-refresh-20191024-114949

**Comment by cockpituous** [comment]
po-refresh in progress on centosci-tasks-w32g0.
Log: https://logs-https-cockpit.apps.ci.centos.org/logs/po-refresh-229-20191024-115757/

**Comment by cockpituous** [comment]
po-refresh  done: https://github.com/cockpituous/cockpit-podman/commits/po-refresh-20191024-115858

**Comment by marusak** [comment]
Ok, this works. It only now [dropped german](https://github.com/cockpituous/cockpit-podman/commit/76f5ecc10a02ac652bff8aba48230df864c28896) because it is not translated to at least 50%.
It also uploaded it https://fedora.zanata.org/iteration/view/cockpit-podman/master?dswid=3189
There is only 112 strings, so when we translate just 62 in one language we can test this properly. Zanata says that translating 62 strings should take 30 minutes. I think it can be even quicker since these strings are very simple (Like `Images`, `Containers`, `Download`..)

**Comment by cockpituous** [comment]
po-refresh in progress on 2-ci-srv-04.
Log: https://209.132.184.41:8493/logs/po-refresh-229-20191024-130101/

**Comment by cockpituous** [comment]
po-refresh  done: https://github.com/cockpituous/cockpit-podman/commits/po-refresh-20191024-130404

**Comment by marusak** [comment]
Dropped German and added Slovak :D 
(and yes, Slovak won't show in Cockpit because it is not in Cockpit itself, but meh)

File: po/de.po [hunk 1]
```diff
@@ -1,26 +0,0 @@
...
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
-"MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=UTF-8\n"
-"Content-Transfer-Encoding: 8bit\n"
-
-#: src/index.html:20
-msgid "Cockpit Starter Kit"
-msgstr "Cockpit Bausatz"
```
Comment (martinpitt) [inline po/de.po]: Haha, I'm glad  it cleaned it up, as that's still the template from starter-kit :grinning:

**Pull request merged**